plugins/cdvdGigaherz/src/SectorConverters.h

```diff
@@ -45,8 +45,10 @@ int Convert<2048,2352>(char *dest, const char* psrc, int lsn)
 	int m = lsn / (75*60);
 	int s = (lsn/75) % 60;
 	int f = lsn%75;
+	unsigned char limit = UCHAR_MAX;
 	unsigned char header[16] = {
-		0x00,0xFF,0xFF,0xFF,0xFF,0xFF,0xFF,0xFF,0xFF,0xFF,0xFF,0x00,   m,   s,   f,0x02
+		0x00,limit,limit,limit,limit,limit,limit,
+		limit,limit,limit,limit,0x00,u8(m),u8(s),u8(f),0x02
```

Comment:
Not keen on this change. The CD standard uses FF when describing the sync field, so IMO it's better to keep using 0xFF here.

Might be better to change the type of `m`, `s`, and `f` to `u8` too instead of doing `u8(m)` and so on?